### PR TITLE
Refresh documentation

### PR DESCRIPTION
**Pull request by seriousben**

### CODE REVIEW

File: specs/provider.yaml [hunk 1]
```diff
@@ -911,7 +946,7 @@ paths:
         pattern: ^[0-9abcdefghjkmnpqrtuvwxyz]{29}$
         format: base32ID
       tags:
-      - Single Sign-On (SSO)
```
Comment (Minivera) [inline specs/provider.yaml]: I'd keep this one, some people don't know SSO stands for Single Sign-on (I certainly didn't :P )

File: specs/provider.yaml [hunk 2]
```diff
@@ -237,18 +275,6 @@ info:
     signature on the canonical form of the request, and that it was signed by
     the public key included in the `X-Signature` header.
 
-    ## Supported languages
```
Comment (Minivera) [inline specs/provider.yaml]: Why remove this?

Comment (seriousben) [reply specs/provider.yaml]: It was moved up

Comment (Minivera) [reply specs/provider.yaml]: 👍 Missed that

File: specs/provider.yaml (hunk 1 above)
Comment (Minivera) [reply specs/provider.yaml]: Waits that's the tag, nevermind, carry one

**Pull request merged**